testing/polipo/APKBUILD

```diff
@@ -0,0 +1,31 @@
+# Contributor: Alexander Georgievskiy <galeksandrp@gmail.com>
+# Maintainer:
+pkgname=polipo
+pkgver=1.1.1
+pkgrel=0
+pkgdesc="The Polipo caching HTTP proxy"
+url="https://github.com/jech/$pkgname"
+arch="all"
+license="MIT"
+depends=""
+makedepends="texinfo"
+install=""
+subpackages=""
+source="$url/archive/$pkgname-$pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$pkgname-$pkgver"
+options="!check"
+
+build() {
+	cd "$builddir"
+
+	make
```

Comment:
No, it’s really not needed, doesn’t have any effect here, because it’s the last command in the function. Please do not spread this cargo cult.